src/main/fc/fc_tasks.c

```diff
@@ -314,6 +315,9 @@ void fcTasksInit(void)
     setTaskEnabled(TASK_SERIAL, true);
 #ifdef BEEPER
     setTaskEnabled(TASK_BEEPER, true);
+#endif
+#ifdef LIGHTS
```

Comment:
This should be `#ifdef USE_LIGHTS` - the `USE_` prefix is used for this type of build flag.

Reply:
OK. I didn't because it is not only a flag it is used to define the pin to use. Maybe best to use two flags: `USE_LIGHTS` and `LIGHTS_PIN` ?

Reply:
Done. Macros to use are now `USE_LIGHTS`, `USE_FAILSAFE_LIGHTS` and `LIGHTS_PIN`.